tests/savefiles.spec.ts

```diff
@@ -1,54 +1,87 @@
 import { readFile } from "fs/promises";
-import { test, expect } from "@playwright/test";
+import { expect, test, type Page } from "@playwright/test";
 
-test("loading a file with length displays the length", async ({ page }) => {
-  await page.goto("/scheme.html?authority=North Somerset");
+let page: Page;
+let json;
+
+test.beforeAll(async ({ browser }) => {
+  page = await browser.newPage();
+  await page.goto("/scheme.html?authority=Adur");
+  // wait for the map to load and interventions panel to appear
+  await page.getByText("Click an object to fill out its attributes").waitFor();
+  // wait for router snapper to load so we can use route tool
+  await page.getByText("Route tool loading...").waitFor({ state: "hidden" });
+
+  json = JSON.parse(
+    await readFile("tests/data/Adur.json", { encoding: "utf8" })
+  );
+});
+
+test.beforeEach(async () => {
```

Comment:
I suspect this'll also be useful for other files, so if we can have a `common.ts` or similar and import, that feels ideal